examples/basic/src/VideoPlayer.android.tsx

```diff
@@ -51,7 +51,7 @@ class VideoPlayer extends Component {
     require('./broadchurch.mp4'),
     {
       description: '(dash) sintel subtitles',
-      uri: 'https://bitmovin-a.akamaihd.net/content/sintel/sintel.mpd',
+      uri: 'Https://bitmovin-a.akamaihd.net/content/sintel/sintel.mpd',
```

Comment:
Incorrect schemes shouldn't be promoted (mixed upper and lower cases).

Although schemes are case-insensitive, the canonical form is lowercase and the implementation should accept uppercase letters as equivalent to lowercase in scheme names (e.g., allow "HTTP" as well as "http") for the sake of robustness.

@see https://www.rfc-editor.org/rfc/rfc3986#section-3.1
@see https://www.rfc-editor.org/rfc/rfc1738#section-2.1

Reply:
The goal is not to promote bad url, but to be able to test it easily

Reply:
I'm saying that because this change is located on the example directory, which, I assume, may be used by some people as examples. For me, it should only contain practical code and if possible, try to enforce good practices. 

@evoactivity: I understand why @freeboub did it that way. In my opinion, it shouldn't be done here, it's not the responsibility of the example to test the code.

Reply:
I would agree with that in general.

With the work @matinzd is doing on #2845 to move to [react-native-test-app](https://github.com/microsoft/react-native-test-app) we should consider separating the examples from the testing apps.

I wouldn't wait on having separate test apps to land this though.

Maybe instead of changing the sintel example we could add another src to the list and give it a more appropriate description?

```js
{
  description: 'allows non-standard casing in scheme'
  uri: 'HttPs://bitmovin-a.akamaihd.net/content/sintel/sintel.mpd',
  type: 'mpd',
},
```
Then once #2845 lads we can clean up the examples for v6.

Reply:
OK, I just remove it

Reply:
I'm ok with @evoactivity's proposal to add a new entry with an appropriate description